docs/graphs/graph-examples/graph-api-example.md

```diff
@@ -0,0 +1,392 @@
+    {'_id': 'lectures/CSC103', 'difficulty': 'hard', '_key': 'CSC103', 'firstname': 'Jean'},
+    {'_id': 'lectures/CSC104', 'difficulty': 'moderate', '_key': 'CSC104', 'firstname': 'Jean'}
+]
+
+url = f"{HTTP_URL}/_api/document/{COLLECTION_NAME_2}"
+resp = session.post(url, data=json.dumps(payload))
+result = json.loads(resp.text)
+print("\nDocuments inserted: ", result)
+
+# Create Edge Collection
+payload = {'name': EDGE_COLL_NAME, "type": 3}
+
+url = f"{HTTP_URL}/_fabric/{GEO_FABRIC}/_api/collection"
+resp = session.post(url, data=json.dumps(payload))
+result = json.loads(resp.text)
+print("\nEdge Collection created: ", result)
+
+payload = [
+    {
+        '_key': 'Jean-CSC101',
+        '_from': 'teachers/Jean',
+        '_to': 'lectures/CSC101',
+        'online': False
+    },
+    {
+        '_key': 'Jean-CSC102',
+        '_from': 'teachers/Jean',
+        '_to': 'lectures/CSC102',
+        'online': True
+    },
+    {
+        '_key': 'Jean-CSC103',
+        '_from': 'teachers/Jean',
+        '_to': 'lectures/CSC103',
+        'online': False
+    },
+    {
+        '_key': 'Bruce-CSC101',
+        '_from': 'teachers/Bruce',
+        '_to': 'lectures/CSC101',
+        'online': True
+    }
+]
+
+url = f"{HTTP_URL}/_api/document/{EDGE_COLL_NAME}"
+resp = session.post(url, data=json.dumps(payload))
+result = json.loads(resp.text)
+print("\nDocuments inserted: ", result)
+
+# Create a Graph
+payload = {
+    "edgeDefinitions": [
+        {
+            "collection": EDGE_COLL_NAME,
+            "from": ["teachers"],
+            "to": ["lectures"]
+        }
+    ],
+    "name": GRAPH_NAME,
+    "options": {}
+}
+
+url = f"{HTTP_URL}/_api/graph"
+resp = session.post(url, data=json.dumps(payload))
+result = json.loads(resp.text)
+print("\nGraph created: ", result)
+
+# Graph Traversal
+# Set `direction` to `out` or `in` for outbound or inbound traversal.
+params = {
+    "vertex": "teachers/Jean",
+    "direction": "out"
+}
+
+url = f"{HTTP_URL}/_api/edges/{EDGE_COLL_NAME}"
+
+resp = session.get(url, params=params)
+result = json.loads(resp.text)
+print("\nGraph traversal: ", result)
+
+# Delete Graph and Collections
+# Set `DELETE_GRAPH` and `DELETE_COLLECTIONS` to `False` if you want to keep the graph and collections.
+if DELETE_GRAPH:
+    params = {"dropCollections": DELETE_COLLECTIONS}
+
+    url = f"{HTTP_URL}/_api/graph/{GRAPH_NAME}"
+
+    resp = session.delete(url, params=params)
+    result = json.loads(resp.text)
+    print("Graph and Collections Deleted: ", result)
+```
+
+  </TabItem>
+  <TabItem value="js" label="JavaScript">
+
+```js
+const https = require("https");
+const httpUrl = "https://api-play.paas.macrometa.io";
+const geoFabric = "_system";
+const apiKey = "your_api_key"; // Replace with your API key
+
+const collectionName1 = "teachers";
+const collectionName2 = "lectures";
+const edgeCollName = "teach";
+const graphName = "lectureteacher";
+
+const deleteGraph = true;
+const deleteCollections = true;
```

Comment:
```suggestion
const deleteCollections = true; // Change to false if you want to keep the collections
```
@diana-macrometa  if you decide to keep it as false instead of true then accordingly change this comment too.